src/output-json-alert.c

```diff
@@ -560,6 +561,14 @@ static int AlertJson(ThreadVars *tv, JsonAlertLogThread *aft, const Packet *p)
                 case ALPROTO_DNS:
                     AlertJsonDns(p->flow, pa->tx_id, jb);
                     break;
+                case ALPROTO_IKEV1:
+                    jb_get_mark(jb, &mark);
+                    if (!JsonIKEV1AddMetadata(p->flow, pa->tx_id, jb)) {
```

Comment:
Suggest `EveIKEV1AddMetadata` -- since moving to JsonBuilder, we're prefixing helper functions with `Eve`.